lib/buildreq.c

```diff
@@ -175,7 +175,7 @@ int rc_aaa_ctx_server(rc_handle * rh, RC_AAA_CTX ** ctx, SERVER * aaaserver,
 
 		result = rc_send_server_ctx(rh, ctx, &data, msg, type);
 
-		if ((result == OK_RC) || (result == CHALLENGE_RC)) {
```

Comment:
I think also the fallback to next server is also independent of REJECT_RC, it happens on TIMEOUT_RC only as far as I understand. So I must have been wrong and this may not be to be explicitly request @martin-belanger what do you think?

Arvind, would you like to add a test case for you change?